src/app/events/events.page.html

```diff
@@ -24,16 +23,16 @@
         >{{event.contacts.region}}
       </p>
       <p *ngIf="event.dates.length > 1" class="event-date">
-        От {{event.dates[0].date}} г. до {{event.dates[event.dates.length -
-        1].date}} г.
+        От {{event.dates[0].startDate | date : 'dd.MM.yyyy'}} г. до {{event.dates[event.dates.length -
+        1].startDate | date :  'dd.MM.yyyy'}} г.
       </p>
-      <p *ngIf="event.dates.length == 1">{{event.dates[0].date}} г.</p>
+      <p *ngIf="event.dates.length == 1" class="event-date">{{event.dates[0].startDate | date :  'dd.MM.yyyy'}} г.</p>
       <app-like-icon class="event-like-btn"></app-like-icon>
     </div>
     <div class="event-info">
       <h3 class="event-title">{{event.shortTitle}}</h3>
       <p class="event-description">{{event.shortDescription}}</p>
-      <button class="more-btn"><p>Прочети повече</p></button>
+      <button class="more-btn" (click)="eventRedirect(event._id)"><p>Прочети повече</p></button>
```

Comment:
why using a function instead of routerLink directly?

Reply:
Because I'm a dumbass and it was from the beginning for a test and I forgot to change it.
Fixed